Lib/multiprocessing/managers.py

```diff
@@ -159,6 +159,11 @@ def __init__(self, registry, address, authkey, serializer):
         self.id_to_local_proxy_obj = {}
         self.mutex = threading.Lock()
 
+    def _track_parent(self):
+        if parent_process := process.parent_process():
```

Comment:
Please use decent syntax...

Reply:
I did not know PEP 572 assignment expressions were out of line for new code?

Reply:
Sorry. I just don't like them :-)

Reply:
Ok. No strong opinion on my end, so I removed it.